### PR TITLE
Merged 'render' command into 'report', maintaining 'render' as a compatibility alias

### PR DESCRIPTION
**Pull request by chrisamin**
Added --from-file and stdin-reading support to 'report', which otherwise was a superset of the 'render' command.
I also took the opportunity to add tests for the file/stdin support, since there wasn't a test for 'render' before. (Which was actually masking a Python 3 error)

### CODE REVIEW

File: tests/base.py [hunk 1]
```diff
@@ -22,8 +22,22 @@
...
+    def __getattr__(self, name):
+        return getattr(self.file_obj, name)
+
+    def isatty(self):
+        return True
+
+
 @contextmanager
-def capture_sys_output():
+def capture_sys_output(use_fake_tty=False):
```
Comment (danielquinn) [inline tests/base.py]: This is awesome.  I'm totally going to steal it.

**Comment by danielquinn** [comment]
It looks fine to me, though I haven't tested it.  You probably need to modify the documentation though to reference the new command instead of the now-alias.

**Comment by astrikos** [comment]
That seems okay to me as well!

**Comment by chrisamin** [comment]
The biggest only-somewhat-related change here is that I'm no longer checking the measurement metadata for the report command, because it was only used to get the "type" of a measurement, which we can always get from the results. It results in output like:

$ ripe-atlas report 3243242342

Error fetching measurement results: [404 Not Found] Not found.

$ ripe-atlas report 1000008

Error fetching measurement results: [403 Forbidden] Authentication credentials were not provided.

$ ripe-atlas report 2001 --probes 99999

There aren't any results for your request.

**Pull request merged**